src/layouts/Admin.js

```diff
@@ -13,7 +13,8 @@ const Admin = (props) => {
   const session = useSelector((state) => state.session);
   // Check whether user is admin or not and update 'isAdmin' value
   // You can check Admin placement and Internship Dashboard with setting isAdmin to true
-  const [isAdmin, setISAdmin] = useState(false)
+  // For now to check, if username is "Admin" in login then we can access Admin Dashboard
+  const [isAdmin, setISAdmin] = useState(session.user.username === "Admin" ? true : false)
```

Comment:
Wait, what?

Reply:
The last line is just for accessing Admin placement and Internship Dashboard by just accessing using username `Admin`, so that it becomes easy to integrate admin APIs and this can be removed later when all the things related to Admin Dashboard are done.